src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/hearingsbyhearingtype/HearingsByHearingTypeReport.java

```diff
@@ -382,20 +386,20 @@ private HearingsByHearingTypeReportDetail getDetail(
             DateListedTypeItem dateListedTypeItem,
             HearingsByHearingTypeCaseData caseData) {
         var detail = new HearingsByHearingTypeReportDetail();
-        detail.setDate(LocalDateTime.parse(
+        detail.setDetailDate(LocalDateTime.parse(
                 dateListedTypeItem.getValue().getListedDate(), OLD_DATE_TIME_PATTERN).toLocalDate().toString());
         detail.setHearingType(hearingTypeItem.getValue().getHearingType());
         detail.setHearingNo(hearingTypeItem.getValue().getHearingNumber());
         var mulRef = StringUtils.defaultString(caseData.getMultipleReference(), "0 -  Not Allocated");
         var subMul = StringUtils.defaultString(caseData.getSubMultipleName(), "0 -  Not Allocated");
         detail.setMultiSub(mulRef + ", " + subMul);
         detail.setCaseReference(caseData.getEthosCaseReference());
-        detail.setLead(Strings.isNullOrEmpty(caseData.getLeadClaimant()) ? "" : "Y");
+        detail.setLead(Strings.isNullOrEmpty(caseData.getLeadClaimant()) ? "N" : "Y");
         detail.setTel(CollectionUtils.isNotEmpty(
                 hearingTypeItem.getValue().getHearingFormat())
                 && hearingTypeItem.getValue().getHearingFormat()
                 .contains("Telephone") ? "Y" : "");
-        detail.setJm("JM".equals(
+        detail.setJm("Yes".equals(
```

Comment:
I think there is a constant for Yes in ecm-common

Reply:
using constant now